source/backend/core/Json/CustomJsonResultFilterAttribute.cs

```diff
@@ -0,0 +1,34 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.Options;
+
+namespace Pims.Core.Json
+{
+    public class NullJsonResultFilter : IResultFilter
```

Comment:
mild preference for filename matching name of the class

Reply:
will fix.